cmd/server.go

```diff
@@ -231,13 +231,15 @@ var Serve = &cobra.Command{
 
 		go jobs.Start()
 
-		events.StartConsumers(db.Gorm, events.Config{
+		events.StartConsumers(db.Gorm, db.Pool, events.Config{
 			UpstreamPush: api.UpstreamConf,
 		})
 
-		playbookRunConsumer := playbook.NewQueueConsumer(db.Gorm, db.Pool)
+		playbookRunConsumer := playbook.NewEventQueueConsumer(db.Gorm, db.Pool)
```

Comment:
Should we name his RunsConsumer or something with `run` in the name  ?

Reply:
I see. And call the `.Listen` inside it?

Reply:
yeah, something like that